e2e/testcafe-devextreme/tests/dataGrid/fixedColumns.ts

```diff
@@ -359,3 +359,58 @@ safeSizeTest('The grid layout should be correct after resizing the window when t
     width: 50,
   }],
 }));
+
+test('DataGrid - Group summary is not updated when a column is fixed on the right side (T1223764)', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const editCell = dataGrid.getDataRow(1).getDataCell(2);
+
+  await t
+    .click(editCell.element)
+    .typeText(editCell.getEditor().element, '11')
```

Comment:
Minor comment.

Let's use the ```{replace: true}``` option of the ```typeText``` to make this test case more obvious
(Now it's a little hard to catch that we will get the ```111``` as edited cell value)

E.g:
```typescript
await t
    .click(editCell.element)
    .typeText(editCell.getEditor().element, '5', { replace: true }) // <---
// other code
    .eql('A: group 0 (Count: 3, Sum of B is 7)'); // <---
```